contracts/itf/ItfLiquidator.sol

```diff
@@ -28,13 +28,18 @@ contract ItfLiquidator {
             MiltonTypes.IporSwapClosingResult[] memory receiveFixedClosedSwaps
         )
     {
-        if (payFixedSwapIds.length > 0) {
-            payFixedClosedSwaps = new MiltonTypes.IporSwapClosingResult[](payFixedSwapIds.length);
-            for (uint256 i = 0; i < payFixedSwapIds.length; i++) {
-                uint256 swapId = payFixedSwapIds[i];
-                IporTypes.IporSwapMemory memory iporSwap = _miltonStorage.getSwapPayFixed(swapId);
+        ItfMilton milton = _milton;
+        IMiltonStorage miltonStorage = _miltonStorage;
+        uint256 payFixedSwapIdsLength = payFixedSwapIds.length;
+        uint256 receiveFixedSwapIdsLength = receiveFixedSwapIds.length;
+        if (payFixedSwapIdsLength > 0) {
+            payFixedClosedSwaps = new MiltonTypes.IporSwapClosingResult[](payFixedSwapIdsLength);
+            uint256 swapId; 
```

Comment:
this 
``` uint256 swapId; ```
can be defined before the if statement, then you don't have to define it in line 56